src/hooks/use-comment-manger.ts

```diff
@@ -0,0 +1,201 @@
+/*!
```

Comment:
there is a type in the file name

Reply:
I don't see any usage of this file and it doesn't seem to be exported out of index to be used anywhere else. Do we still needed?

Reply:
we use it in https://github.com/Atypon-OpenSource/manuscripts-article-editor/blob/5804e9252fb2d1451419ea26fd563699772ca61d/src/components/projects/CommentList.tsx#L34

Reply:
ah, It was collapsed for me. Can you please fix the typo? "-manger" to "...-manager"

Reply:
updated